CHANGELOG.md

```diff
@@ -25,6 +26,115 @@ The below sections record the changes for each API version in each release as fo
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/QualityOnDemand/blob/r3.1/code/API_definitions/quality-on-demand.yaml)
+
+### Added
+
+* Added HTTP-422 error response when qos profile is not applicable during session creation by @maxl2287 in https://github.com/camaraproject/QualityOnDemand/pull/433
+* Added error response documentation in `info.description` by @hdamker in https://github.com/camaraproject/QualityOnDemand/pull/467
+
+### Changed
+
+* Adjusted x-correlator pattern as defined in CAMARA Commonalities by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/465
+* Updated documentation regarding device identifiers in responses and added DeviceResponse object to limit device identifiers in responses to exactly one by @eric-murray in https://github.com/camaraproject/QualityOnDemand/pull/453
+
+### Fixed
+
+* Resolved mismatch between test and API definition for networkAccessIdentifier and sinkCredential by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/462
+
+### Removed
+
+* Removed AUTHENTICATION_REQUIRED error by @eric-murray in https://github.com/camaraproject/QualityOnDemand/pull/436
+* Removed the IDENTIFIER_MISMATCH error code option from 422 responses and associated test cases by @eric-murray in https://github.com/camaraproject/QualityOnDemand/pull/453
+
+## qos-profiles v1.1.0-rc.2
+
+**qos-profiles v1.1.0-rc.2 is the first release candidate of the version 1.1.0**
+
+qos-profiles v1.1.0 will be a minor update of the API, and is backward compatible with v1.0.0.
+
+- API definition **with inline documentation**:
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/r3.1/code/API_definitions/qos-profiles.yaml&nocors)
+  - [View it on Swagger Editor](https://camaraproject.github.io/swagger-ui/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/r3.1/code/API_definitions/qos-profiles.yaml)
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/QualityOnDemand/blob/r3.1/code/API_definitions/qos-profiles.yaml)
+
+### Added
+
+* Added an optional countryAvailability field to the QoS profile to allow an API provider to indicate in which countries (and networks) the profile is available by @eric-murray in https://github.com/camaraproject/QualityOnDemand/pull/435
+* Added error response documentation in `info.description` by @hdamker in https://github.com/camaraproject/QualityOnDemand/pull/467
+
+### Changed
+
+* Adjusted x-correlator pattern as defined in CAMARA Commonalities by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/465
+* Updated documentation regarding device identifiers in responses and added DeviceResponse object to limit device identifiers in responses to exactly one by @eric-murray in https://github.com/camaraproject/QualityOnDemand/pull/453
```

Comment:
As far as I checked DeviceResponse is not managed in qos-profile API

Reply:
Good point, the changes are in the same PR across the three APIs, but here the text doesn't fit.

Have change it to "Updated documentation regarding multiple device identifiers in requests and the expected behavior of implementations" in https://github.com/camaraproject/QualityOnDemand/pull/463/commits/9518a165ac8c2a6a6dbafa2e85fc1f8343883f50